client/src/main/java/com/alibaba/nacos/client/config/impl/ClientWorker.java

```diff
@@ -418,7 +422,7 @@ private String null2defaultGroup(String group) {
     /**
      * Check config info.
      */
-    public void checkConfigInfo() {
+    private void checkConfigInfo() {
```

Comment:
if you remove checkInfo, how do you handle doing dynamic task creation

Reply:
'dynamic task'. I can't understand it,can you explain it